integration/commands_administration_test.go

```diff
@@ -912,6 +923,83 @@ func TestCommandsAdministrationServerStatusMetrics(t *testing.T) {
 	}
 }
 
+func TestCommandsAdministrationServerStatusStress(t *testing.T) {
+	// TODO This tests reproduces the problem with SQLSTATE 3F000: https://github.com/FerretDB/FerretDB/issues/1317
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1317")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	client := collection.Database().Client()
+
+	dbNum := runtime.GOMAXPROCS(-1) * 10
+
+	ready := make(chan struct{}, dbNum)
+	start := make(chan struct{})
+
+	var wg sync.WaitGroup
+	for i := 0; i < dbNum; i++ {
+		wg.Add(1)
+
+		go func(i int) {
+			defer wg.Done()
+
+			ready <- struct{}{}
+
+			<-start
+
+			dbName := fmt.Sprintf("%s_stress_%d", collection.Database().Name(), i)
+			db := client.Database(dbName)
+			_ = db.Drop(ctx) // make sure DB doesn't exist (it will be created together with the collection)
+
+			collName := fmt.Sprintf("stress_%d", i)
+
+			schema := fmt.Sprintf(`{
+				"title": "%s",
+				"description": "Create Collection Stress %d",
+				"primary_key": ["_id"],
+				"properties": {
+					"_id": {"type": "string"},
+					"v": {"type": "string"}
+				}
+			}`, collName, i,
+			)
+			opts := options.CreateCollectionOptions{
+				Validator: bson.D{{"$tigrisSchemaString", schema}},
+			}
+
+			// Attempt to create a collection for Tigris with a schema.
+			// If we get an error, that's MongoDB (FerretDB ignores that argument for non-Tigris handlers),
+			// so we create collection without schema.
+			err := db.CreateCollection(ctx, collName, &opts)
+			if err != nil {
+				if strings.Contains(err.Error(), `support for field "validator" is not implemented yet`) {
```

Comment:
> FerretDB ignores that argument for non-Tigris handlers

That comment directly contradicts the code

Reply:
Yeah, because the current PR had been started earlier than these changes https://github.com/FerretDB/FerretDB/commit/68364a7605e02d9c544e3fb2035d6a44a25723c7#diff-a0885bdda838486091e4d1e879cac1cf239ac08bacf84468825f746bc72b035c were merged. The comment could have better wording (even though the current wording is also yours 😅).